README.md

```diff
@@ -148,6 +148,7 @@ check-keys             | REDIS_EXPORTER_CHECK_KEYS            | Comma separated
 check-single-keys      | REDIS_EXPORTER_CHECK_SINGLE_KEYS     | Comma separated list of keys to export value and length/size, eg: `db3=user_count` will export key `user_count` from db `3`. db defaults to `0` if omitted.  The keys specified with this flag will be looked up directly without any glob pattern matching.  Use this option if you don't need glob pattern matching;  it is faster than `check-keys`.
 check-streams          | REDIS_EXPORTER_CHECK_STREAMS         | Comma separated list of stream-patterns to export info about streams, groups and consumers. Syntax is the same as `check-keys`.
 check-single-streams   | REDIS_EXPORTER_CHECK_SINGLE_STREAMS  | Comma separated list of streams to export info about streams, groups and consumers. The streams specified with this flag will be looked up directly without any glob pattern matching.  Use this option if you don't need glob pattern matching;  it is faster than `check-streams`.
+count-keys             | REDIS_EXPORTER_COUNT_KEYS            | Comma separated list of patterns to count, eg: `db3=sessions:*` will count all keys with prefix `sessions:` from db `3`. db defaults to `0` if omitted.
```

Comment:
Can you add a brief comment here that it uses SCAN to count the keys? so users are aware of the methodology? Maybe also a warning that using this option can still add addtl load for large key sets (all matching key values are shipped to the exporter, right?)